src/pageObject.js

```diff
@@ -1,15 +1,42 @@
 'use strict';
 
 class Header {
+  constructor() {
+    this._logo = 'Conduit logo';
+  }
 
+  get logo() {
+    return this._logo;
+  }
 };
 
 class PageObject {
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  }
 
+  clickOnLogo() {
+    return 'Click on the ' + this.header.logo;
+  }
 };
 
 class ArticlePage extends PageObject {
+  super(url, slug) {
+    this.url = url + slug;
```

Comment:
Call the parent class's constructor using 'super(url)' before assigning the 'url' property in the ArticlePage class constructor. 🚀